decayAmplitude/productionVertex.h

```diff
@@ -76,6 +76,9 @@ namespace rpwa {
 
 		virtual std::string name() const { return "productionVertex"; }  ///< returns label used in graph visualization, reporting, and key file
 
+		virtual double getTprime() const {printWarn<<"getTprime not overwritten"<<std::endl;return 0.;};
+
+
```

Comment:
Since productionVertex could be much more than diffractiveDissociation, thinking e.g. of Deck-like effects, an abstract function would i.m.o. not be the optimal choice, since t' is not very meaningful for these kind of processes, and thus a corresponding errorMessage would have to be implemented everytime.
I would change the message though:
printWarn<<"getTprime not defined for current process "<<name()<<std::endl;